api/controllers/userController.js

```diff
@@ -0,0 +1,17 @@
+const userService = require('../services/userService');
+const { catchAsync } = require('../middlewares/error');
+const { detectError } = require('../util/detectError');
+
+const kakaoLogin = catchAsync(async (req, res) => {
+  const kakaoToken = req.headers.authorization;
+
+  if (!kakaoToken) detectError('NONE_ACCESS_TOKEN', 401);
```

Comment:
해당 에러메시지는 인가 과정에서 accedToken이 없을 경우에 반환되는 에러메시지와 동일하여 혼란을 줄 수 있을 것 같습니다.
카카오 유저 정보를 받기 위해 토큰을 받지 못했다는 의미를 전달하는 것은 어떨까요?
```suggestion
  if (!kakaoToken) detectError('NONE_KAKAO_TOKEN', 401);
```